File: A2Z_sheet/Array/Easy/04_remove_duplicates/solution.go
func removeDuplicates(nums []int) int {
    k := 0
    for i := 1; i<len(nums); i++ {
        if(nums[i] != nums[k]){
            k += 1;
            nums[k] = nums[i];
        }
    }
    return k + 1;
}

File: A2Z_sheet/Array/Easy/04_remove_duplicates/solution.ts
function removeDuplicates(nums: number[]): number {
    let k: number = 0;
    for(let i: number =1;i<nums.length;i++){
        if(nums[i] != nums[k]) nums[++k] = nums[i];
    }
    return k + 1;
};
